File: julia/seed.jl
using DuckDB

con = DBInterface.connect(DuckDB.DB, "../db/database.duckdb")

# create tchp tables
DBInterface.execute(con, "INSTALL tpch")
DBInterface.execute(con, "LOAD tpch" )
DBInterface.execute(con, "CALL dbgen(sf = 10)")
DBInterface.close(con)

File: python/seed.py
import duckdb

if __name__ == "__main__":
    # Connect to the DuckDB database
    con = duckdb.connect("../db/database.duckdb")

    # Create TPCH tables
    con.execute("INSTALL tpch")
    con.execute("LOAD tpch")
    con.execute("CALL dbgen(sf=10)")

    # Close the connection
    con.close()
